src/03_modules.py

```diff
@@ -5,20 +5,23 @@
 level operating system functionality.
 """
 
+import os
 import sys
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+s = sys.argv
+list_args = [x for x in s]
+print("args passed: " + str(list_args))
```

Comment:
When passing args in from the command line, they will already be in a list. Consider how you would print each one one per line. This should be run from the command line with several arguments after it to test it out.